syntax = "proto3";

package core_system.product.v1;

import "messages/common/request.proto";
import "messages/product/response.proto";

option py_generic_services = true;

// ProductService 정의
service ProductService {
    rpc GetProductsByShop(core_system.common.v1.GetByShopRequest) returns (GetProductsByShopResponse);
}
